curriculum/challenges/english/25-front-end-development/lecture-introduction-to-javascript-libraries-and-frameworks/6734e86f590727c5e7e9ec5e.md

```diff
@@ -78,7 +73,7 @@ State determines the styling of UI components.
 
 ### --feedback--
 
-Review the beginning the video where this was discussed. 
+Review the section where this was discussed. 
```

Comment:
same here 

Let's make this more specific. like "reviewing the beginning section" or something like that. Apply to all feedbacks